index.html

```diff
@@ -12,11 +12,34 @@
 
 <body>
     <header>
-        <div class="mobile">
-            <a href="#"> Ikpro </a>
-            <nav>
-                <a href="#"><i class="fa-solid fa-bars"></i></a>
-            </nav>
+        <div >
+            <div class="mobile flex1">
+                <a href="#"> Ikpro </a>
+                <nav>
+                    <a href="#" id="hamburgerButton"><i class="fa-solid fa-bars"></i></a>
+                </nav>
+            </div>
+            
+            <ul id="contactList" class="disappear ">
+                <li class="mobile flex1">
+                    <a href="#">  </a>
+                    <nav>
+                        <a href="#" id="hamburgerButton2"><h3>&times;</h3></a>
+                    </nav>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#Projects" class="options"> <h3 class="mobile-menu">Portfolio</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#About" class="options"> <h3 class="mobile-menu">About</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+                <li class="mobile flex1 menu-option">
+                   <a href="#contact" class="options"> <h3 class="mobile-menu">Contact</h3> </a>
+                    <a href="#" class="arrow"><i class="fa-solid fa-angle-right"></i></a>
+                </li>
+            </ul>
```

Comment:
- [ ] the navigation links are working fine in the mobile version , however they aren't working in the desktop version so please try to fix that